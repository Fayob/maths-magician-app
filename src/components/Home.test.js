import React from 'react';
import { render } from '@testing-library/react';

import Home from './Home';

describe('test for Home Component', () => {
  it('should render Home Component', () => {
    expect(render(<Home />)).toMatchSnapshot();
  });
});
