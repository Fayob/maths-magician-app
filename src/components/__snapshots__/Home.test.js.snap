// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for Calculator Component should render Calculator Component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="calculator"
      >
        <h4>
          Let\`s do some math!
        </h4>
        <div
          class="container"
        >
          <div
            class="header"
          >
            0
          </div>
          <div
            class="AC"
          >
            AC
          </div>
          <div
            class="addSub"
          >
            +/-
          </div>
          <div
            class="percent"
          >
            %
          </div>
          <div
            class="divide"
          >
            รท
          </div>
          <div
            class="multiply"
          >
            x
          </div>
          <div
            class="minus"
          >
            -
          </div>
          <div
            class="plus"
          >
            +
          </div>
          <div
            class="equal"
          >
            =
          </div>
          <div
            class="one"
          >
            1
          </div>
          <div
            class="two"
          >
            2
          </div>
          <div
            class="three"
          >
            3
          </div>
          <div
            class="four"
          >
            4
          </div>
          <div
            class="five"
          >
            5
          </div>
          <div
            class="six"
          >
            6
          </div>
          <div
            class="seven"
          >
            7
          </div>
          <div
            class="eight"
          >
            8
          </div>
          <div
            class="nine"
          >
            9
          </div>
          <div
            class="zero"
          >
            0
          </div>
          <div
            class="dot"
          >
            .
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="calculator"
    >
      <h4>
        Let\`s do some math!
      </h4>
      <div
        class="container"
      >
        <div
          class="header"
        >
          0
        </div>
        <div
          class="AC"
        >
          AC
        </div>
        <div
          class="addSub"
        >
          +/-
        </div>
        <div
          class="percent"
        >
          %
        </div>
        <div
          class="divide"
        >
          รท
        </div>
        <div
          class="multiply"
        >
          x
        </div>
        <div
          class="minus"
        >
          -
        </div>
        <div
          class="plus"
        >
          +
        </div>
        <div
          class="equal"
        >
          =
        </div>
        <div
          class="one"
        >
          1
        </div>
        <div
          class="two"
        >
          2
        </div>
        <div
          class="three"
        >
          3
        </div>
        <div
          class="four"
        >
          4
        </div>
        <div
          class="five"
        >
          5
        </div>
        <div
          class="six"
        >
          6
        </div>
        <div
          class="seven"
        >
          7
        </div>
        <div
          class="eight"
        >
          8
        </div>
        <div
          class="nine"
        >
          9
        </div>
        <div
          class="zero"
        >
          0
        </div>
        <div
          class="dot"
        >
          .
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`test for Home Component should render Home Component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="home_component"
      >
        <h3>
          Welcome to our page!
        </h3>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum
        </p>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="home_component"
    >
      <h3>
        Welcome to our page!
      </h3>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum
      </p>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
